cvs diff nsFileLocations.cpp (in directory D:\Projects\mozilla\xpfe\appshell\src\)
Index: nsFileLocations.cpp
===================================================================
RCS file: /cvsroot/mozilla/xpfe/appshell/src/nsFileLocations.cpp,v
retrieving revision 1.57
diff -r1.57 nsFileLocations.cpp
280,281c280,281
<     // set its directory an aunt of the executable.
<     nsSpecialSystemDirectory cwd(nsSpecialSystemDirectory::OS_CurrentProcessDirectory);
---
>     // Use the Moz_BinDirectory
>     nsSpecialSystemDirectory cwd(nsSpecialSystemDirectory::Moz_BinDirectory);

*****CVS exited normally with code 1*****

